packages/editor/src/components/post-card-panel/index.js

```diff
@@ -100,6 +122,7 @@ export default function PostCardPanel( { className, actions } ) {
 					>
 						{ title ? decodeEntities( title ) : __( 'No Title' ) }
 					</Text>
+					<ActionTrigger item={ post } action={ viewPostAction } />
```

Comment:
I think the end goal is to unify the actions and handle them holistically. Since we don't do that yet and consumers can also pass other actions, I think it makes more sense to rename `ActionTrigger` to `ViewPostLink` or something similar for now.